### DISCUSSION TITLE
run flet at endeavouros linux

### DISCUSSION THREAD

**Discussion by nexblu** [flet-dev/flet#3032] [General]
excuse me sir, i have error when i run flet app and I'm confused about how to solve it

I've tried installing MPV
https://archlinux.org/packages/extra/x86_64/mpv/

the error message like this: 
```
/home/adit/.flet/bin/flet-0.22.0/flet/flet: error while loading shared libraries: libmpv.so.1: cannot open shared object file: No such file or directory```

**Comment by ndonkoHenri** [comment]
Please can you try the suggestions here:
- https://github.com/flet-dev/flet/issues/2823
- https://github.com/flet-dev/flet/issues/2637

**Reply by nexblu** [reply]
okay thanks that worked

**Reply by ndonkoHenri** [reply]
Can you please share a link to the comment/suggestion which helped?

**Reply by nexblu** [reply]
https://github.com/flet-dev/flet/issues/2823#issue-2173706942

I tried this
![image](https://github.com/flet-dev/flet/assets/141644469/1a866e4f-03ca-4f28-b16c-1ef70a377bd8)
